### ISSUE TITLE
All capped keys should be at the top.

### ISSUE THREAD

**Issue by jdalton** [mthadley/eslint-plugin-sort-destructure-keys#300]
If Capitalized keys are at the top, I assumed all caps would be higher than them but they are injected as if not all capped.

**Comment by mthadley** [comment]
Yep, I think I'd have the same expectation if I was destructuring a mix of camel and all caps casing, which I somehow never found myself doing. Thanks for noticing!

Something I'll look into changing in the near term, though if someone else gets to it first, that'd also be great.
